### PR TITLE
Add props and useState

### PR DESCRIPTION
**Pull request by berlinum**
Add HSL color scheme
Add props.color and useState

### CODE REVIEW

File: src/components/PasswordInput.js [hunk 1]
```diff
@@ -2,15 +2,32 @@ import React from "react";
...
-  width: 250px;
-  height: 30px;
-  font-size: 18px;
-  border: solid 3px red;
+  width: 270px;
+  height: 35px;
+  font-size: 20px;
+  border: solid 2px coral;
   border-radius: 10px;
+  background-color: hsl(${props => props.color}, 60%, 50%);
```
Comment (lmachens) [inline src/components/PasswordInput.js]: `color` is not the color. it's the `hue`

File: src/components/PasswordInput.js [hunk 2]
```diff
@@ -2,15 +2,32 @@ import React from "react";
...
 `;
 
-function PasswordInput({ value }) {
-  return <Input type="password" value={value} />;
+function PasswordInput({ color }) {
+  const [value, setValue] = React.useState("");
+  color = () => {
```
Comment (lmachens) [inline src/components/PasswordInput.js]: don't forget the `const`.
why is this a function? the name `color` doesn't sound like a function.

File: src/components/PasswordInput.js [hunk 3]
```diff
@@ -2,15 +2,32 @@ import React from "react";
...
+  font-size: 20px;
+  border: solid 2px coral;
   border-radius: 10px;
+  background-color: hsl(${props => props.color}, 60%, 50%);
 `;
 
-function PasswordInput({ value }) {
-  return <Input type="password" value={value} />;
+function PasswordInput({ color }) {
+  const [value, setValue] = React.useState("");
```
Comment (lmachens) [inline src/components/PasswordInput.js]: The PasswordInput component is not controlled anymore. This makes it difficult to set the value from parent component.

Comment (berlinum) [reply src/components/PasswordInput.js]: You're right... Sorry, ich pushed a "draft version" but you were so quick and reviewed it before I fixed it  ☺️

**Pull request merged**